src/Commands/ProjectCreateCommand.php

```diff
@@ -351,6 +352,20 @@ function ($state) use ($siteDir, $source) {
                 ->provider($this->git_provider)
                 ->provider($this->site_provider)
 
+            ->progressMessage('Initialize build-providers.json')
+            ->taskPushbackSetup()
+                ->dir($siteDir)
+                ->provider($this->git_provider, $this->ci_provider)
+            ->progressmessage('Set build secrets')
+            ->addCode(
+                function ($state) use ($site_name) {
+                    $secretValues = [
+                        'token' => $this->git_provider->token($this->git_provider->tokenKey())
+                    ];
+                    $this->writeSecrets("{$site_name}.dev", $secretValues, false, '.build-secrets/tokens.json');
```

Comment:
`.build-secrets` should be enforced in the `secrets` API methods, not inserted by the caller.

Reply:
Got it.